src/frontend/js/widgets/SyllabusCourseRunsList/components/CourseRunEnrollment/index.tsx

```diff
@@ -69,6 +70,11 @@ const messages = defineMessages({
     description: 'Helper text in the enroll button for non logged in users',
     id: 'components.CourseRunEnrollment.loginToEnroll',
   },
+  courseRunStartIn: {
+    defaultMessage: 'The course starts {relativeStartDate}',
```

Comment:
Why not use future ?

```suggestion
    defaultMessage: 'The course will start {relativeStartDate}',
```

Reply:
Because I saw it already used with this `{relativeStartDate}` pattern elsewhere in the codebase: https://github.com/search?q=repo%3Aopenfun%2Frichie+%22The+course+starts%22&type=code

Reply:
OK!